### DISCUSSION TITLE
Version 0.9.8

### DISCUSSION THREAD

**Discussion by anametologin** [anametologin/krohnkite#77] [Announcements]
1. New layout `Columns`. 
-support resize
-support moving windows left/right up/down by krohnkite shortcuts
-support dragging inside layout and among monitors

2. New dragging way (only columns layout support for now) you can see where the target window will be

3. moving from es5 to es6

<hr /><em>This discussion was created from the release <a href='https://github.com/anametologin/krohnkite/releases/tag/'>Version 0.9.8</a>.</em>

**Comment by rmelendezz** [comment]
Thank you so much for all of your hard work, honestly using KDE alongside Krohnkite is just amazing. If there is a way to buy you a few cups of coffees please do let me know.
